### DISCUSSION TITLE
Mulch System Webview

### DISCUSSION THREAD

**Discussion by rsssxd** [Divested-Mobile/DivestOS-Build#52] [Q&A]
Mulch System Webview is present on the device. Is it ok to use Mulch browser as an extra?

**Comment by SkewedZeppelin** [comment]
(sent as e-mail but didn't work?)
I mean I do maintain both if it isn't obvious.
Mulch (standalone browser) is more secure than Mull but doesn't offer the same
privacy benefits like strict per-site data isolation and advanced content
blocking (uBlock Origin).

You might consider keeping both installed.

**Comment by rsssxd** [comment]
You misunderstood me. Do I just need to use both at the same time?

![Screenshot_20220215-145528_Droid-ify_1](https://user-images.githubusercontent.com/60405617/154057712-414b5e83-af03-4d8f-b750-e18f3fd8f730.png)

**Reply by SkewedZeppelin** [reply]
You do not need both.
Mulch WebView is effectively mandatory on DivestOS.
Mulch standalone is completely optional.

**Comment by rsssxd** [comment]
Is it possible for me to use Mulch System Webview as a standalone browser?

**Reply by SkewedZeppelin** [reply]
No.
The WebView is for apps to use.
If you want a standalone Chromium you can use Mulch or Bromite.

**Reply by Tachi107** [reply]
I believe that you could try using one of the "WebView browsers" that you can find on the DivestOS website: https://divestos.org/index.php?page=browsers#webview
